src/pages/blog.tsx

```diff
@@ -1,101 +1,61 @@
-import { Container, Divider, Flex, Text, chakra } from '@chakra-ui/react';
-import { NextPage } from 'next';
-import Image from 'next/image';
-import React from 'react';
-import veltBusMan from '../assets/man-with-bus-background.jpg';
-import veltBoss from '../assets/velt-boss.jpg';
+import { Flex } from '@chakra-ui/react';
+import { createClient } from 'next-sanity';
 import { Layout } from '../components/layout';
+import { PortableText } from '@portabletext/react';
 
-export const Hero = chakra(Image, {
-  baseStyle: {
-    objectFit: 'cover',
-    width: '100%',
-    maxWidth: 'container.xl',
-    borderRadius: '2xl'
-  },
-  shouldForwardProp: prop =>
-    ['width', 'height', 'src', 'alt', 'fill', 'quality'].includes(prop),
+const postQuery = `*[_type == "post"] {
+    _id,
+    author,
+    headline,
+    "image": Image[]{
+      _type,
+      asset,
+      attribution,
+      caption,
+    },
+    "articleBody": articleBody[]{
+      _key,
+      _type,
+      "children": children[] {
+        _key,
+        _type,
+        text,
+        marks,
+      },
+      markDefs[],
+      style,
+      asset,
+      listItem,
+      level,
+    },
+    }`;
+
+const Blog = ({ posts }) => {
+  const [post] = posts;
+  return (
+    <Layout>
+      <Flex flexDirection="column">
+        <PortableText value={[post]} />
```

Comment:
Where did the hero go? And the author?

Reply:
I was thinking we do that in another PR as this was just connect sanity to website